src/elements/content-uploader/ContentUploader.js

```diff
@@ -1049,6 +1049,7 @@ class ContentUploader extends Component<Props, State> {
 
         const hasFiles = items.length !== 0;
         const isLoading = items.some(item => item.status === STATUS_IN_PROGRESS);
+        const allDone = items.every(item => item.progress === 100);
```

Comment:
Maybe should be `isDone`?

Reply:
It seems like we should be using `item.status` rather than checking the progress percentage. Can we use `!isLoading` or do we need to introduce a new status enum value?

Reply:
The problem here is that, when the progress is 100, the status remains in IN_PROGRESS for a few seconds, which in this time user can click 'cancel' button but the file has been already uploaded.

Reply:
> Maybe should be `isDone`?

Sure

Reply:
I may be missing something. It sounds like we'd definitely want to use `!isLoading` if the progress property can stay at 100 even while it's not finished. Let's sync up tomorrow.